code/headers/st7735_inverted_color.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <display_adapter.hpp>
+#include <hwlib.hpp>
+#include <st7735.hpp>
+
+
+namespace r2d2::display {
+
+    template <std::size_t CursorCount, class DisplayScreen>
+    class st7735_inverted_color_c
```

Comment:
One possible way to `fix` this is to have this be a decorator for either the buffered or unbuffered version and to only invert the `data` parameter of the `set_pixel` functions, and then call the original ones with the modified color parameter. This would mean the implementation of the inverted classes in no longer necessary.

Reply:
since the inverted color set registers in the display the classes are needed although we know inherit from the buffer/unbufferd and eliminate 1 class in between also the inverted color class is almost empty now since we only need to set those registers in the constructor and use all functions from the "normal" implementation